### PR TITLE
Add post on statistical significance and its misconceptions

### PR DESCRIPTION
**Pull request by Morgane-des-Ligneris**
I am making this pull request at the last minute, sorry, but if there are mistakes and problems, I will correct them after the presentation.

### CODE REVIEW

File: collections/_posts/2024-11-21-Statistical_significance.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+# References
+[^1]: M. Schmidt, C.F. Christiansen, F. Mehnert, K.J. Rothman, H.T. Sørensen. Non-steroidal anti-inflammatory drug use and risk of atrial fibrillation or flutter: population based case-control study. BMJ, 343 (2011), p. d3450
+[^2]: T.-F. Chao, C.-J. Liu, S.-J. Chen, K.-L. Wang, Y.-J. Lin, S.-L. Chang, _et al._ The association between the use of non-steroidal anti-inflammatory drugs and atrial fibrillation: a nationwide case–control study. Int J Cardiol, 168 (2013), pp. 312-316
+[^3]: Morten Schmidt, Kenneth J. Rothman. Mistaken inference caused by reliance on and misinterpretation of a significancetest, International Journal of Cardiology, Volume 177, Issue 3, 2014, Pages 1089-1090, ISSN 0167-5273, https://doi.org/10.1016/j.ijcard.2014.09.205.
```
Comment (nathanpainchaud) [inline collections/_posts/2024-11-21-Statistical_significance.md:85]: Missing space:
```suggestion
[^3]: Morten Schmidt, Kenneth J. Rothman. Mistaken inference caused by reliance on and misinterpretation of a significance test, International Journal of Cardiology, Volume 177, Issue 3, 2014, Pages 1089-1090, ISSN 0167-5273, https://doi.org/10.1016/j.ijcard.2014.09.205.
```

**Review by nathanpainchaud — CHANGES_REQUESTED** [review]
Just 2 typos that caught my eye. Otherwise fantastic work on summarizing a not so simple topic :wink:

File: collections/_posts/2024-11-21-Statistical_significance.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+The paper advocates to stop using the concept of statistical significance. They do not advocate to ban the P-value, but rather to stop using the conventional dichotomous way to take the decision if a results refutes or supports an hypothesis. 
+# A call for better science practice 
+They propose that the **pre-registration of studies** and the **commitment to publish all analyses and results** it a solution to reduce the to present issue on the reliance of 'significance'. There still can be bias that can occur even with the best intention, but it is still better than to ignore the issue. Also we should Interpret results broadly. It’s fine to suggest reasons for findings, but we should discuss multiple explanations, not just preferred ones. Focusing on factors like study design, data quality, and mechanisms, often matter more than P values or intervals.
```
Comment (bayaca) [inline collections/_posts/2024-11-21-Statistical_significance.md:61]:
```suggestion
They propose that the **pre-registration of studies** and the **commitment to publish all analyses and results** as a solution to reduce the reliance on 'significance'. There still can be bias that can occur even with the best intention, but it is still better than to ignore the issue. Also we should interpret results broadly. It’s fine to suggest reasons for findings, but we should discuss multiple explanations, not just preferred ones. Focusing on factors like study design, data quality, and mechanisms, often matter more than P values or intervals.
```

**Review by bayaca — CHANGES_REQUESTED** [review]
small changes on a sentence that was probably rewritten.
great post !

**Review by bayaca — APPROVED** [review]

**Review by nathanpainchaud — APPROVED** [review]

**Pull request merged**